### PR TITLE
Update unit tests and minor tweaks

### PR DESCRIPTION
**Pull request by jrichter1**

### CODE REVIEW

File: browser/services/data.js [hunk 1]
```diff
@@ -15,6 +13,7 @@ class InstallerDataService {
     this.downloading = false;
     this.installing = false;
 
+    let env = require('remote').require('../main/env');
```
Comment (kenfinnigan) [inline browser/services/data.js]: Is this move purely to make the declaration of 'env' closer to its use or another reason?

Comment (jrichter1) [inline browser/services/data.js]: Yes, there is another reason - when I tried to import the class into the tests I got the following:
```Cannot find module 'remote'```
This seems to have solved it.

**Comment by kenfinnigan** [comment]
Is this PR needed anymore?

**Comment by jrichter1** [comment]
The updated unit tests should still be relevant, but I need to change them again. I can either do it as a part of this PR, or I can close this and create another one when I'm finished.

**Comment by kenfinnigan** [comment]
Whatever is easier for you is fine

**Pull request closed without merging**